README.md

```diff
@@ -79,15 +79,25 @@ If you click on the icon which looks like a set of gears, you should see an inte
 
 ![Rhasspy settings](rhasspy/images/Rhasspy_settings.PNG)
 
-It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You will also have to fill out the device information underneath the Audio Recording and Audio Playing settings. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. You can test the audio recording and playback by clicking on the icon that looks like a house.
+It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. A download button should appear at the top of the page like below if aditional data is needed.
+
+![image](https://github.com/wyattschool/sandman/assets/77877259/505852bd-9a8d-4181-b9b4-bb2e95cb26bb)
+
+You will also have to fill out the device information underneath the Audio Recording and Audio Playing settings. Clcik the green colored area with the carret to the left to expose the microphone/Audio Recording or speaker/Audio Playing settings. Once expanded you will need to select the correct available devices. For instance the audio recording device might be "Default Audio Device (sysdefault:CARD=Device_1)" and the audio playing device might be "Default Audio Device (sysdefault:CARD=Device)". It might be helpful to plug in one device at a time and review which device has been added to the list. The list is long so this may take some trial and error. 
+
+![image](https://github.com/wyattschool/sandman/assets/77877259/df1a6414-691f-4806-9e08-ae1efc74d92b)
+
+You can test the audio recording and playback on home page for Rhasspy aka the house button on the left navigation bar. On the home page use the speak button and text box to test the speaker. Click the Wake Up button then speak into the microphone to see if commands are recognized like "Lower back". This should activate a relay for a few seconds as well as provide verbal feedback. You may also want to set the volume at this point. 1 is the max volume.
```

Comment:
I would change use the speak button and text box to be something like "put some test text into the text field next to the Speak button and then press the Speak button to test the speaker." I think that's a bit more clear. Also because we haven't set up the sentences yet, we won't get recognition from the phrase you have given. Additionally, if you're not running Sandman at the time of setting this up (which is likely the case unless you're doing this out of order), there will not be any feedback from Sandman or relay feedback either. So I think that part should be removed and instead it there is a way to replay the recorded audio that is also on the homepage.

Reply:
I agree with what you are saying and you make good points about the recognition of the phrases at this point in setup.